purchase_order_no_owner/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,12 @@
+This module adds no_owner field in the purchase order,
+allowing users to indicate that the ordered inventory is not associated with any specific owner.
+
+Background
+~~~~~~~~~~
+
+users may unintentionally overlook filling out the owner_id field when creating a purchase order
+for inventory items that should be linked to a specific owner and subsequently confirm the order.
+Once the PO confirmed, users will not able to edit the owner_id field to fill the correct owner.
+
+This module is designed to address this situation by Implementing a process that blocks saving the order or confirming it if the owner_id is blank.
+Additionally, when the order is not associated with any specific owner, users have the option to set the no_owner field to "True" instead.
```

Comment:
This is not part of the background, therefore move it to the main part of the description or USAGE.rst.